Minor fixes of variables naming

dolamroth opened the pull request.

dolamroth committed:
Change BaseUserMixin to BaseUser

dolamroth committed:
Rename schema_request/response to request/response_schema

dolamroth committed:
Remove redundant json-deserialization in WS endpoint

dolamroth committed:
Minor fixes to type annotations

DmitryBurnaev commented on starlette_web/common/authorization/base_user.py:
```diff
@@ -1,10 +1,10 @@
-class BaseUserMixin:
+class BaseUser:
```
не совсем согласен с неймингом( это же не базовая модель от которой можно унаследоваться и получишь готовую модель для юзера.

dolamroth replied:
Откатил изменения, оставил комментарий о том, почему это миксин

dolamroth committed:
Revert renaming BaseUserMixin, leave a note about a reason

dolamroth committed:
Fix rare obscure warnings/bugs in Windows

dolamroth committed:
Wrap closing functions for redis/postgres with asyncio.Task with explicit

Follows a recommendation from Python docs:

https://docs.python.org/3/library/asyncio-task.html#asyncio.shield

Save a reference to tasks passed to this function, to avoid a task disappearing mid-execution. The event loop only keeps weak references to tasks. A task that isn’t referenced elsewhere may get garbage collected at any time, even before it’s done.
